CorsixTH/Src/th_gfx_sdl.cpp

```diff
@@ -307,7 +313,9 @@ bool render_target::create(const render_target_creation_params* pParams) {
   }
 
   Uint32 iRendererFlags =
-      (pParams->present_immediate ? 0 : SDL_RENDERER_PRESENTVSYNC);
+      (pParams->present_immediate
+           ? 0
+           : SDL_RENDERER_PRESENTVSYNC | SDL_RENDERER_ACCELERATED);
```

Comment:
SDL_RENDERER_ACCELERATED should be the preferred default anyway. What did applying this flag here do?

Reply:
This was likely driver's experimentation to attempt to apply shaders. It is not necessary and does not affect this. I've removed it.